EIPS/eip-7329.md

```diff
@@ -0,0 +1,245 @@
+---
+eip: 7329
+title: ERC/EIP Repository split
+description: Split the ERC specifications out of the EIP repository into a new repository, so that only core protocol EIPs remain
+author: Lightclient (@lightclient), Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-erc-repo-split/15053
```

Comment:
Optional: feel free to consider use existing EIP/ERC split FEM thread https://ethereum-magicians.org/t/proposal-forking-ercs-from-eips-repository/12804

@shemnon

Reply:
That didn't show up on my searches.  Thanks.